### ISSUE TITLE
New rtables adopters feedback (ongoing collection)

### ISSUE THREAD

**Issue by ayogasekaram** [insightsengineering/rtables#579]
Pharma Collaboration team newly onboarded to rtables for table creation have expressed the following as helpful. rtables documentation should call-out, highlight, or add the following information.

[ ] Handling of missing values and `df_explicit_na` behaviour
[ ] Meaning and placement of the different footnote types + `fnotes_at_path` use
[ ] `var_relabel` instead of mutating in pre-processing

**Comment by shajoezhu** [comment]
that is right, @ayogasekaram , can you create an issue for "df_explicit_na" in tern please. Thanks!

**Comment by Melkiades** [comment]
I think the `fnotes_at_path` needs expanded documentation. I cannot find it quickly and it is missing something describing how you can do more complicated pathing there
